docs/source/common_issues.rst

```diff
@@ -504,12 +504,19 @@ to see the types of all local variables at once. Example:
    #     a: builtins.int
    #     b: builtins.str
 .. note::
-
-   ``reveal_type`` and ``reveal_locals`` are only understood by mypy and
-   don't exist in Python. If you try to run your program, you'll have to
-   remove any ``reveal_type`` and ``reveal_locals`` calls before you can
-   run your code. Both are always available and you don't need to import
-   them.
+   ``reveal_type`` and ``reveal_locals`` are understood by mypy during
+   typechecking, and don't have to be imported as functions. However,
+   if you don't import them, then they don't exist at runtime! Therefore,
+   you'll have to remove any ``reveal_type`` and ``reveal_locals`` calls
+   from your program or else Python will give you an error at runtime about
+   those names being undefined. Alternatively, you can import ``reveal_type``
+   from ``typing_extensions`` (or, in more recent versions of python, from
```

Comment:
`reveal_type` was added in 3.11, and we're now awaiting 3.14, so it isn't some shiny new thing. Maybe swap these parts? 

```
Alternatively, you can import ``reveal_type`` from ``typing`` (or, in versions of python older than 3.11, from ``typing_extensions``)
```

Reply:
I like it, thanks.